src/ert/_c_wrappers/enkf/analysis_config.py

```diff
@@ -210,6 +207,27 @@ def minimum_required_realizations(self) -> int:
     def have_enough_realisations(self, realizations) -> bool:
         return realizations >= self.minimum_required_realizations
 
+    @property
+    def case_format(self) -> Optional[str]:
```

Comment:
There seems to be a problem annotated in the initialization:
doing `analysis_iter_config=AnalysisIterConfig()` makes just one `AnalsysIterConfig` for all defaulted `AnalysisConfig`. need to use:

```python
__init__(
analysis_iter_config=None,
....
):
  if analysis_iter_config is None:
      analysis_iter_config = AnalysisIterConfig()
```

Reply:
Very good catch! I will include the fix as a separate commit to this pr